client/src/modules/ProductOverview/Header.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+class Header extends React.Component {
+    constructor() {
+      super();
+    }
```

Comment:
remember React Component constructors take a `props` input, and so does `super()`